### DISCUSSION TITLE
How to import a differenct package?

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#18605] [Questions]
![image](https://user-images.githubusercontent.com/31676163/215335358-0c60b071-7af9-4177-8229-5e90858cef24.png)

I was trying to use emailJs but it seems nuxt3 can't find the module, how to do it though?

**Comment by Baroshem** [comment]
Hey, 

Was this emailjs added as a line of code from its docs or was it typed by you with IDE path autocompletion?

Also, what is the error when you hover over the package name?
